### PR TITLE
feat(source): update redhat-stage temp source to production source

### PR DESCRIPTION
**Pull request by jasinner**
Testing of Red Hat RPM security data.

See https://github.com/google/osv.dev/issues/1404

### CODE REVIEW

File: source_test.yaml [hunk 1]
```diff
@@ -256,15 +256,18 @@
...
-- name: 'redhat-stage'
+- name: 'redhat'
   versions_from_repo: False
-  type: 1
-  bucket: 'osv-test-cve-osv-conversion'
-  directory_path: 'redhat-temporary'
+  rest_api_url: 'https://security.access.redhat.com/data/osv/all.json'
+  type: 2
+  directory_path: 'data/osv'
+  detect_cherrypicks: False
```
Comment (jasinner) [inline source_test.yaml:264]: I don't know what this setting does.

File: source_test.yaml [hunk 2]
```diff
@@ -256,15 +256,18 @@
...
   ignore_patterns: ['^(?!RH[BES]{1}A-).*$']
   extension: '.json'
   db_prefix: ['RHBA-', 'RHEA-', 'RHSA-']
-  link: 'https://storage.googleapis.com/osv-test-cve-osv-conversion/'
+  ignore_git: False
```
Comment (jasinner) [inline source_test.yaml:268]: It's not a git repo so probably irrelevant?

File: source_test.yaml [hunk 3]
```diff
@@ -256,15 +256,18 @@
...
+  human_link: 'https://access.redhat.com/errata/{{ BUG_ID }}'
+  link: 'https://security.access.redhat.com/data/osv/'
```
Comment (jasinner) [inline source_test.yaml:270]: This link returns 404 because we don't have a index.html setup. Does it matter?

**Review by andrewpollock — APPROVED** [review]

File: source_test.yaml (hunk 2 above)
Comment (andrewpollock) [reply source_test.yaml:268]: It is, but it's also harmless.

File: source_test.yaml (hunk 3 above)
Comment (andrewpollock) [reply source_test.yaml:270]: I was just [sanity checking that](https://github.com/google/osv.dev/blob/bb583e85b054e3afec31fc52b5f7ab57cf2cf6f0/gcp/appengine/frontend3/src/templates/vulnerability.html#L35). I don't expect this to be a problem, but it's something to review post-import.

File: source_test.yaml (hunk 1 above)
Comment (andrewpollock) [reply source_test.yaml:264]: It's only relevant for [analysing Git repositories](https://github.com/google/osv.dev/blob/fe6155f7cfa0e5df0ae1ef20c7b16f5c20bebed1/osv/impact.py#L120), and I don't believe the records being published have any Git commit ranges on them, and this is generally the correct approach as far as I understand cherry picking and the analysis of them...

**Pull request merged**